Runtime/Core/MeshHandles.cs

```diff
@@ -211,13 +223,112 @@ internal static void CreatePointBillboardMesh(List<Vector3> points, Mesh mesh)
         /// Draw a set of vertices.
         /// </summary>
         static void BuildVertexMeshInternal(ProBuilderMesh mesh, Mesh target, IEnumerable<int> indexes)
-
         {
             target.Clear();
             target.name = "pb_ElementGraphics::PointMesh";
             target.vertices = mesh.positionsInternal;
             target.subMeshCount = 1;
             target.SetIndices(indexes as int[] ?? indexes.ToArray(), MeshTopology.Points, 0);
         }
+
+        internal static void CreateEdgeBillboardMesh(ProBuilderMesh mesh, Mesh target)
+        {
+            target.Clear();
+            const ushort k_MaxPointCountUShort = ushort.MaxValue / 4;
+            var lineCount = mesh.edgeCount;
+
+            target.indexFormat = lineCount > k_MaxPointCountUShort
+                ? Rendering.IndexFormat.UInt32
+                : Rendering.IndexFormat.UInt16;
+
+            var vertices = mesh.positionsInternal;
+
+            Vector3[] positions = new Vector3[lineCount * 4];
```

Comment:
This can be improved by keeping a static reused list and the new mesh API. I didn't do it as a part of this PR because it would require a separate path for pre-2019.3 code.